src/JS/display.js

```diff
@@ -0,0 +1,29 @@
+const displayMovies = (data, elementId) => {
```

Comment:
kindly change the name of the argument from `elementId` to `sectionId`, to be consistent with the logic of your code.

Reply:
Ok I will Change it.